src/adapters/__tests__/commonTests.js

```diff
@@ -55,16 +55,16 @@ export default () => [
             migrations: schemaMigrations({ migrations: [] }),
           }),
       ).not.toThrowError()
```

Comment:
```suggestion
      ).not.toThrow()
```
Same on line 45 and 48

Reply:
you mean whitespace or the `toThrow` → `toThrowError`?

Reply:
Whitespace change was a mistake, sorry about that. I was talking about changing all `toThrowError` into `toThrow`

Reply:
> agree with the linter that consistency is more important

Yeah, but something must have gone horribly wrong. Right now there is `toThrow` and `toThrowError` _in the same file_ just few lines of each other. There had to be something wrong with how eslint if as you said it were to replace all names with primary name instead of aliases.

In the _next line_ there is `toThrow` changed from `toThrowError`  That is why I highlighted this line to fix it.

Reply:
🤔 ok — will take a look, didn't notice that